web/packages/teleport/src/Roles/RoleEditor/StandardEditor/standardmodel.ts

```diff
@@ -979,39 +998,105 @@ function ruleToModel(
   conversionErrors: ConversionError[];
 } {
   const { resources = [], verbs = [], where = '', ...unsupported } = rule;
+
   const conversionErrors = unsupportedFieldErrorsFromObject(
     pathPrefix,
     unsupported
   );
+
   const resourcesModel = resources.map(
     // Resource kind can be unknown, so we just create a necessary option on
     // the fly.
     k => resourceKindOptionsMap.get(k) ?? { label: k, value: k }
   );
-  const verbsModel = verbs.map(v => verbOptionsMap.get(v));
-  const knownVerbsModel: VerbOption[] = [];
-  verbsModel.forEach((verb, i) => {
-    if (verb !== undefined) {
-      knownVerbsModel.push(verb);
-    } else {
-      conversionErrors.push({
-        type: ConversionErrorType.UnsupportedValue,
-        path: `${pathPrefix}.verbs[${i}]`,
-      });
+
+  const allVerbs = verbs.includes('*');
+  const verbsModel = allowedVerbsForResourceKinds(resources).map(verb => ({
+    verb,
+    checked: allVerbs, // Mark all as checked if there's a wildcard.
+  }));
+
+  if (allVerbs) {
+    // If there's a wildcard, it needs to be the only verb. Other combinations
+    // are not supported because of the editor UI structure.
+    if (verbs.length > 1) {
+      conversionErrors.push(
+        unsupportedValueWithReplacement(`${pathPrefix}.verbs`, '["*"]')
```

Comment:
Should this error mention that if wildcard  value cannot be combined with other verbs.?

Reply:
I don't want to add much more to this PR, but I'll add a TODO to consider it in future if more such cases would benefit from clarifications.